packages/auth0/src/handlers/auth0-handlers.ts

```diff
@@ -162,28 +162,8 @@ export const createAuth0Handlers = (store: Auth0Store, people: Iterable<Person>,
           expires_in: 86400,
           token_type: "Bearer",
         });
-      } catch (error: any) {
-        let assertCondition = 'Assert condition failed: ';
-        if (error?.message?.startsWith(assertCondition)) {
-          let errorMessage = error.message.slice(assertCondition.length);
-          let errorCode =
-            errorMessage.slice(3, 5) === '::'
-              ? parseInt(errorMessage.slice(0, 3))
-              : 500;
-          let errorResponse = errorMessage.slice(5);
-          res.status(errorCode).send(errorResponse);
-        } else {
-          console.error(error)
-          res
-            .status(500)
-            .json({
-              error: {
-                name: error.name,
-                message: error.message,
-                stack: error.stack,
-              },
-            });
-        }
+      } catch (error) {
+        next(error);
```

Comment:
Do we need to have a try/catch here at all now? Will any thrown errors be automatically handled by the default error handler?

Reply:
(in voice chat) It seems that we need it in here. We might be able to deal with it more elegantly, but roll with it for now.